conformance/src/main/scala/za/co/absa/enceladus/conformance/interpreter/DynamicInterpreter.scala

```diff
@@ -45,32 +44,16 @@ object DynamicInterpreter {
     * @return The conformed dataframe
     *
     */
-  def interpret(conformance: ConfDataset,
-                inputDf: Dataset[Row],
-                jobShortName: String = "Conformance",
+  def interpret(conformance: ConfDataset, inputDf: Dataset[Row], jobShortName: String = "Conformance",
                 experimentalMappingRule: Boolean)
```

Comment:
In this case it actually can be re-expanded and it just fits into the function length limit.

But I suggest we really revisit our method length limit in  `scalastyle_config.xml`. In a 8 parameter function just the declaration takes 16% of the quote then.

Reply:
- there is loose hanging `experimentalMappingRule` because it wouldn't fit. Please return to the param per line.
- parameters should not and I think are not counted into the length of the method. And having 8 parameters is a bit of an overkill, but that is for another topic and time.

Reply:
My wrong, function declaration is not counted into the limit. Please consider my previous comment moot. 😊